homeassistant/components/gogogate2/sensor.py

```diff
@@ -0,0 +1,94 @@
+"""Support for Gogogate2 garage Doors."""
+from typing import Callable, List, Optional
+
+from gogogate2_api.common import AbstractDoor, get_configured_doors, get_door_by_id
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import DEVICE_CLASS_BATTERY
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .common import (
+    DeviceDataUpdateCoordinator,
+    cover_unique_id,
+    get_data_update_coordinator,
+)
+from .const import DOMAIN
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: Callable[[List[Entity], Optional[bool]], None],
+) -> None:
+    """Set up the config entry."""
+    data_update_coordinator = get_data_update_coordinator(hass, config_entry)
+
+    async_add_entities(
+        [
+            DoorSensor(config_entry, data_update_coordinator, door)
+            for door in get_configured_doors(data_update_coordinator.data)
+            if door.sensorid and door.sensorid != "WIRE"
```

Comment:
It would be nice to make `WIRE` a constant ..... and move it into the `pypi` in a future PR.